src/apiClient.ts

```diff
@@ -42,9 +67,33 @@ class ApiClient {
     })
   }
 
-  public setTokens(tokens: Tokens): void {
-    this.tokens.accessToken = tokens.accessToken
-    this.tokens.refreshToken = tokens.refreshToken
+  public getConfiguration() {
+    return {
+      ...this.configuration,
+      configured: Object.keys(this.configuration).length > 0,
+      version: this.version,
+    }
+  }
+
+  public setAccessToken(accessToken: { token: string; expires: Date }): void {
+    if (!this.runsInBrowser) {
+      throw new Error(
+        "You may only globally set and access-token on a client side usage, to avoid leaking the token on the global module for a server side implementation. Please pass tha access-token as an option to the request instead when using the package server side!"
```

Comment:
```suggestion
        "You may only globally set (xyz) and access-token on a client side usage, to avoid leaking the token on the global module for a server side implementation. Please pass the access-token as an option to the request instead when using the package server side!"
```